seg_lapa/config/train.yaml

```diff
@@ -5,6 +5,7 @@ defaults:
   - trainer: trainer
   - scheduler: disabled
   - logger: wandb
+  - checkpoint_callback: checkpoint_callback
```

Comment:
Please restructure the code to follow format in early stopping PR. I've created methods to retrieve a list of callbacks. Add this model checkpoint callback to the callbacks/standard.yaml file.